pkgs/by-name/bl/bluesnooze/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  fetchurl,
+  lib,
+  stdenv,
+  unzip,
+}:
+stdenv.mkDerivation rec {
```

Comment:
Let's use `finalAttrs` to improve override flexibility:
- #315337

Reply:
Oh that's a great idea, I've bumped into this problem many times when making overrides. Thanks!